docs/cli.md

```diff
@@ -59,6 +59,11 @@ sb deploy [--docker-image]
 
 #### Examples
 
+Deploy default image on current GPU node:
```

Comment:
'current GPU node' => 'specified nodes'? 
'current' here looks a little inexact.

Reply:
this is a specific example running on `localhost`, so it only means current node

Reply:
changed to "local GPU node"